packages/next-app/src/components/Sidebar.tsx

```diff
@@ -25,14 +25,16 @@ function useSidebarContext() {
 export default function Sidebar({
   children,
   isFloating = false,
+  padding = "py-5 px-4",
```

Comment:
You could also give a name more true to what the function of this variable is and simply call this `classNames` and add it to the end of the `cn`.